nihruk/design-system
# Add a Nunjucks macro for the Footer component

## How to test
- Navigate to https://design-system-git-footer-macro-nihruk.vercel.app/component/footer
- Confirm the footer examples all look good and work well
- Confirm the Github and npm social media icon links in the footer of every page on the documentation site look good and work well
